src/configs/esbuild.es5.config.js

```diff
@@ -81,12 +82,14 @@ module.exports = (folder, globalName) => {
         },
       }),
     ],
+    minifyWhitespace: true,
+    minifyIdentifiers: true,
+    minifySyntax: false,
```

Comment:
Can we make minification configurable similar to the rollup version?
https://github.com/rdkcentral/Lightning-CLI/blob/62bd1234f57d3be7b2442faf58d95e57d10a9d50/src/configs/rollup.es6.config.js#L63-L64

Reply:
Sure. Please check now..